docs/source/examples/setup_your_own_data.ipynb

```diff
@@ -0,0 +1,862 @@
+{
```

Comment:
<p>Perhaps this should be its own notebook? I guess it fits here since it's just pulling the MPI runs from the catalogue. But something like this would be perfect for a multi-model DCPP notebook example. That might be another PR though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/climpred/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>its the same ipynb file but in the docs it appears as two different subheadings.</p>